nxviz/io.py

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from networkx import Graph, MultiGraph
+
+
+def graph_from_dataframe(
+        dataframe,
+        threshold_by_percent_unique=0.1,
+        threshold_by_count_unique=None,
+        node_id_columns=[],
+        node_property_columns=[],
+        edge_property_columns=[],
+        collapse_edges=True,
+        edge_agg_key='weight'):
+    '''
+    Build an undirected graph from a pandas dataframe.
+    This function attempts to infer which cells should become nodes
+    based on either:
+        a) what percentage of the column are unique values (defaults to 10%)
+        b) an explicit count of unique values
+            (i.e. any column with 10 unique columns or less)
+        c) an explicit list of column keys
+            (i.e. ['employee_id', 'location_code'])
+    '''
```

Comment:
Please add sphinx-style docstrings to document what each parameter is.

Example to get you started:

```
:param dataframe: A pandas dataframe containing the data to be converted into a graph.
:param float threshold_by_percent_unique: ...
...
:param bool collapse_edges: Graphs are instantiated as `MultiGraph` objects. Setting this to True will convert the `MultiGraph` into a `Graph` object.
...
```

This ensures that the API doc is complete.